src/util/CLIValidator.hpp

```diff
@@ -0,0 +1,51 @@
+//
+// Created by Carolin on 19.05.2020.
+//
+
+#ifndef KICLIENT_CLIVALIDATOR_HPP
+#define KICLIENT_CLIVALIDATOR_HPP
+
+#include <arpa/inet.h>
+
+namespace CLI::validate {
+
+    class StringLengthTwo : public Validator {
+        public:
+            StringLengthTwo() : Validator("StrLengthTwo") {
+                func_ = [](std::string &str) {
+                    if (str.length() < 2) {
+                        return std::string("String has to contain min. two characters: ") + str + ')';
+                    }
+                    return std::string();
+                };
+            }
+    };
+
+    class IPAddress : public Validator {
+        public:
+            IPAddress() : Validator("IPV4 or IPV6") {
+                func_ = [](std::string &str) {
```

Comment:
Hier gebe ich sonar recht, das könnte const sein.
```suggestion
                func_ = [](const std::string &str) {
```